contracts/JBDirectory.sol

```diff
@@ -319,4 +331,42 @@ contract JBDirectory is IJBDirectory, JBOperatable {
 
     emit SetPrimaryTerminal(_projectId, _token, _terminal, msg.sender);
   }
+
+  /** 
+    @notice
+    The owner (Juicebox multisig) can add addresses which are allowed to change
+    a project's controller. Those addresses are known and vetted controllers as well as
+    contracts designed to launch new projects. This is not a requirement for all controllers.
+    However, unknown controllers may require additional transactions to perform certain operations.
+
+    @dev
+    If you would like your JBController allowlisted, please reach out to the Juicebox dev team.
```

Comment:
If you would like an address/contract allowlisted

^ remember these are no longer necessarily controllers